### PR TITLE
Add outgoing transfers filter to v0.9.37 branch

### PR DESCRIPTION
**Pull request by ghzlatarev**
* static filter for outgoing xcm transfers by currency id
* added tests\
* related to https://github.com/Manta-Network/Manta/pull/1103

### CODE REVIEW

File: xtokens/src/lib.rs [hunk 1]
```diff
@@ -466,6 +480,12 @@ pub mod module {
 				currencies.len() <= T::MaxAssetsForTransfer::get(),
 				Error::<T>::TooManyAssetsBeingSent
 			);
+			for (currency_id, _) in currencies.iter() {
+				ensure!(
+					!T::OutgoingAssetsFilter::contains(currency_id),
+					Error::<T>::AssetDisabledForOutgoingTransfers
+				);
+			}
```
Comment (ferrell-code) [inline xtokens/src/lib.rs]: I think this loop is unneeded.

Line# 506 checks this below

File: xtokens/src/lib.rs [hunk 2]
```diff
@@ -99,6 +99,8 @@ pub mod module {
 		/// MultiLocation filter
 		type MultiLocationsFilter: Contains<MultiLocation>;
 
+		type OutgoingAssetsFilter: Contains<Self::CurrencyId>;
```
Comment (Dengjianping) [inline xtokens/src/lib.rs:103]: Can you add a comment here?

File: xtokens/src/lib.rs (hunk 1 above)
Comment (ghzlatarev) [reply xtokens/src/lib.rs]: oh yeah i moved it in the next loop, but forgot to delete this. thx

Comment (Dengjianping) [reply xtokens/src/lib.rs]: agreed

**Review by Dengjianping — APPROVED** [review]

**Review by ferrell-code — APPROVED** [review]

**Pull request merged**